compiler/rustc_codegen_llvm/src/gotoc/cbmc/goto_program/symtab_transformer/transformer.rs

```diff
@@ -0,0 +1,742 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use super::super::{
+    BinaryOperand, CIntType, DatatypeComponent, Expr, ExprValue, Location, Parameter, SelfOperand,
+    Stmt, StmtBody, SwitchCase, Symbol, SymbolTable, SymbolValues, Type, UnaryOperand,
+};
+use crate::btree_map;
+use num::bigint::BigInt;
+use std::collections::{BTreeMap, HashSet};
+
+/// The `Transformer` trait is a visitor pattern for the `SymbolTable`.
+/// To use it, you just need to implement the three symbol table accessor methods,
+/// and then override any methods that you want to change the behavior of.
+///
+/// The entry point is `transform_symbol_table`. It then:
```

Comment:
Nit: replace "it" with "the transformer" or similar specific item.

Reply:
Fixed!